data/mods/Aftershock/items/armor/modular_exos/modular_exoskeleton_modules.json

```diff
@@ -0,0 +1,406 @@
+    },
+    "power_draw": 1500,
+    "revert_to": "exo_small_carry",
+    "use_action": {
+      "ammo_scale": 0,
+      "target": "exo_small_carry",
+      "msg": "You deactivate the module.",
+      "menu_text": "Turn off module",
+      "type": "transform"
+    }
+  },
+  {
+    "id": "exo_flashlight",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "mounted lamp (off)", "str_pl": "mounted lamps (off)" },
+    "description": "This is a light that can be secured to an exoskeleton hardpoint.  It has two levels of brightness.",
+    "weight": "500 g",
+    "volume": "500 ml",
+    "material": [ "steel", "plastic" ],
+    "symbol": "o",
+    "color": "white",
+    "charges_per_use": 1,
+    "ammo": [ "battery" ],
+    "flags": [ "USE_UPS", "CANT_WEAR", "EXO_SMALL_GADGET" ],
+    "use_action": {
+      "type": "transform",
+      "msg": "You turn the lamp on low.",
+      "target": "exo_flashlight_low",
+      "active": true,
+      "need_charges": 1,
+      "need_charges_msg": "There is not enough charge to activate the module."
+    }
+  },
+  {
+    "id": "exo_flashlight_low",
+    "copy-from": "exo_flashlight",
+    "type": "TOOL",
+    "name": { "str": "mounted lamp (low)", "str_pl": "mounted lamps (low)" },
+    "power_draw": 1000,
+    "revert_to": "exo_flashlight",
+    "use_action": {
+      "ammo_scale": 0,
+      "menu_text": "Turn to high",
+      "type": "transform",
+      "msg": "You turn the heavy duty flashlight off.",
+      "target": "exo_flashlight_high"
+    },
+    "flags": [ "LIGHT_250", "USE_UPS", "CANT_WEAR", "EXO_SMALL_GADGET" ]
+  },
+  {
+    "id": "exo_flashlight_high",
+    "copy-from": "exo_flashlight",
+    "type": "TOOL",
+    "name": { "str": "mounted lamp (high)", "str_pl": "mounted lamps (high)" },
+    "power_draw": 2000,
+    "revert_to": "exo_flashlight",
+    "use_action": {
+      "ammo_scale": 0,
+      "menu_text": "Turn off",
+      "type": "transform",
+      "msg": "You turn the heavy duty flashlight off.",
+      "target": "exo_flashlight"
+    },
+    "flags": [ "LIGHT_500", "USE_UPS", "CANT_WEAR", "EXO_SMALL_GADGET" ]
+  },
+  {
+    "id": "exo_hearing",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str_sp": "audio protection (off)" },
+    "description": "This is an audio protection exoskeleton module.  While active, it also protects your hearing from loud noises.",
+    "weight": "500 g",
+    "volume": "500 ml",
+    "material": [ "steel", "plastic" ],
+    "symbol": "o",
+    "color": "white",
+    "charges_per_use": 1,
+    "ammo": [ "battery" ],
+    "flags": [ "USE_UPS", "CANT_WEAR", "EXO_HELMET_GADGET", "ONLY_ONE" ],
+    "use_action": {
+      "type": "transform",
+      "msg": "You activate the helmet's audio protection.",
+      "target": "exo_hearing_on",
+      "active": true,
+      "need_charges": 1,
+      "need_charges_msg": "There is not enough charge to activate the module."
+    }
+  },
+  {
+    "id": "exo_hearing_on",
```

Comment:
Doesnt seem to be working.

Reply:
This targeted the wrong item, corrected now.

Reply:
The only active effect is the "PARTIAL_DEAF", like electronic hearing protection. I'd like to duplicate the hearing CBM's effects to complete this item.